tg_bot/modules/rss.py

```diff
@@ -0,0 +1,319 @@
+import html
+
+from telegram import ParseMode, constants
+from telegram.ext import CommandHandler
+
+from tg_bot import dispatcher, SUDO_USERS, updater
+from tg_bot.modules.sql import rss_sql as sql
+from tg_bot.modules.helper_funcs.chat_status import bot_admin, user_admin
+
+from feedparser import parse
+
+
+def show_url(bot, update, args):
+    tg_chat_id = str(update.effective_chat.id)
+
+    if len(args) >= 1:
+        # there is an actual link written
+
+        tg_feed_link = args[0]
+        link_processed = parse(tg_feed_link)
+
+        if 'title' in link_processed.feed:
```

Comment:
Wouldn't it look a bit cleaner if all the `feed_{title,description,link}` and family be initialised as `'Unknown'` already so we can just skip all the else stuff 🤔